tests/stubtest_allowlists/win32.txt

```diff
@@ -8,6 +8,8 @@ mmap.PAGESIZE
 # alias for a class defined elsewhere,
 # mypy infers the variable has type `(*args) -> DupHandle` but stubtest infers the runtime type as <class DupHandle>
 multiprocessing.reduction.AbstractReducer.DupHandle
+# Multiprocessing.popen_fork exists on Windows but fails to import
+multiprocessing.popen_fork
```

Comment:
Can you move this allowlist entry down to line 60 or so, so it's next to all the other modules that either don't exist or fail to import on Windows?

Reply:
Sure, I had put it here so it was next to the other multiprocessing entry in the file

Reply:
Yeah, but this section of the allowlist is for things that are _potentially_ fixable, either by changing the typeshed stub or by improving mypy/stubtest somehow. This entry doesn't belong in this section, as it's not something that's fixable; it's just a thing that we need stubtest to ignore on Windows :)